default/AI/AIDependencies.py

```diff
@@ -26,6 +26,8 @@
                   fo.planetSize.gasGiant: -1
                   }
 
+SUPPLY_MOD_SPECIALS = {'WORLDTREE_SPECIAL':{-1:1}}
```

Comment:
Can you add comment to code about this dict values? Looks like -1 is value that added to other planet sizes.

IMHO adding function `get_planet_supply_bonus(planet: fo.planet) -> int` will be more readable.